src/components/containers/Characters.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import DeckOfCards from '../DeckOfCards/DeckOfCards';
+
+export default class Characters extends Component {
+  static propTypes = {
+    count: PropTypes.number.isRequired,
+  };
+
+  state = {
+    page: 1,
+    items: []
+  };
+
+  getCharacters = (count, page) => {
```

Comment:
This 100% works, but it's more conventional to extract fetching functions to a separate file in a "services" folder. This is useful when you get into larger projects and need to reuse fetching functions across multiple components.